app/index.js

```diff
@@ -79,8 +81,11 @@ if (!gotTheLock) {
   app.on("ready", handleAppReady);
   app.on("quit", () => console.debug("quit"));
   app.on("render-process-gone", onRenderProcessGone);
-  app.on("will-quit", () => {
+  app.on("will-quit", async () => {
     console.debug("will-quit");
+    if (mqttClient) {
+      await mqttClient.disconnect();
+    }
```

Comment:
for future improvement, we might want to put this events inside the mqttClient. That way we can handle them better. For example, I think we should also handle this in quit, and probably if the configuration changes (if we want to allow to dynamically enable mqtt). But future improvement, not needed for now.